how-to/offline_cpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Offline SOS CPUs except BSP before launch UOS
```

Comment:
> Offline

Would it be helpful to offer the ability to online the previously-offlined CPUs (maybe a cli arg)?

Reply:
ACRN hypervisor needs unused physical cpus to start guest VMs. If we bring back the cpus that is, make them available for service OS then hypervisor will not be able to launch new guest VMs. So it is better not to bring the cpus online back for Kata use case.